0000-template-process.md

```diff
@@ -0,0 +1,99 @@
+---
+process: (fill me in with a unique ident, my_new_process)
+start-date: (fill me in with today's date, YYYY-MM-DD)
+author: (name of the main author)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+modifies:
+ - [RFC 0000 my_old_process]()
+category: process
+---
+
+<!--
+If you are proposing to change a process with regard of how the nix community
+conducts, then use this template. Some examples are, without being an exhaustive list:
+
+- Change the RFC process, the organization of the issue tracker or the forum
+- Change community workflows or other comunity infrastructure
+- Amend the code of conduct and similar high level normative documents
+-->
+
+# Summary
+[summary]: #summary
+
+<!-- One paragraph to resume this document (motivation and future process). -->
+
+In order to address "X", I/we propose to "Y".
+
+# Classification
+[classification]: #classification
+
+<!-- Please check the relevant boxes (typically one) -- or add your own. -->
+
+- [ ] general habits and decision making
+- [ ] core technical protocols & processes
+- [ ] contributer efficiency support
+
+# Motivation
+[motivation]: #motivation
+
+<!-- What's wrong? Please feel encouraged to benchmark us against other
+(open source or other) ecosystems. -->
+
+# Current Process
+[as-is]: #current-process
+
+<!-- Describe the current process as it is observed out in the wild.
+If there has been a previous RFC for this process, please mention it,
+but prefer the state of the world "as-is". In a final paragraph, please
+describe its shortcomings and how they relate to the motivation.
+
+Make use of BPMN 2.0 notation, if you'd find that useful. -->
```

Comment:
BPMN 2.0 notation?

Reply:
I'll clarify that, `BPMN 2.0` is a specific imppementation (one of the more widely used) to graphically denote processes (= "process diagrams").

Reply:
How well does this integrate into markdown/github?

Reply:
Reading through https://gitlab.com/gitlab-org/gitlab/-/issues/22365, it might be that https://kroki.io/ is a suitable answer that at least circles around your question.

I think it makes sense to put a practical suggestion here on how to include such a diagram. Processes without diagrams are like bread without butter.